app/views/dashboards/following_tags.html.erb

```diff
@@ -54,3 +60,36 @@
     </div>
   </div>
 </main>
+
+<script>
+  document.getElementById('follows_update_form').addEventListener('submit', checkChanged)
+  
+  function addChanged(element) {
+    element.setAttribute('changed', true);
+  };
+
+  function checkChanged(event) {
+    if (document.querySelector('input[changed]')) {
+      disableAllUnchanged();
+    }
+    else {
+      event.preventDefault();
+    }
+  };
+
+  function disableAllUnchanged() {
+    document.querySelectorAll('div[id^="follows"]').forEach(disableUnchanged)
+  };
+
+  function disableUnchanged(item) {
+    item = item.getElementsByTagName('input')
+    var id = item[0]
+    var point = item[1]
```

Comment:
Ideally we use ES6 here and not use `var`. I think re-defining `item` to something more descriptive and not erasing the argument is best practice. :)